packages/eslint-plugin-kolibri/lib/rules/vue-no-unused-translations.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview Disallow unused translation string definitions.
+ */
+
+'use strict';
+
+const eslintPluginVueUtils = require('eslint-plugin-vue/lib/utils');
+
+const utils = require('../utils');
+const constants = require('../constants');
+
+const { GROUP_$TRS } = constants;
+
+const $TR_FUNCTION = '$tr';
+
+const create = context => {
+  let hasTemplate;
+  let definitionNodes = [];
+  let usedStrings = [];
+
+  const initialize = {
+    Program(node) {
+      if (!utils.checkVueEslintParser(context)) {
+        return;
+      }
+      hasTemplate = Boolean(node.templateBody);
+    },
+  };
+
+  const scriptVisitor = Object.assign(
+    {},
+    // Check for a use within an $tr() function - a 'true use' of the string.
+    {
+      'CallExpression[callee.type="MemberExpression"]'(node) {
+        if (node.callee.property.name == $TR_FUNCTION && node.arguments.length) {
+          node.arguments.forEach(arg => usedStrings.push(arg.value));
+        }
+      },
+    },
+    // This will include any defined Identifiers (eg, coachesLabel or submitAction) that is
+    // found inside of an array or as the value assigned in an object. This covers the case
+    // where a string may be used dynamically or in an iterator.
+    {
+      ObjectExpression(node) {
+        node.properties.forEach(prop => {
+          if (prop.value.value) {
```

Comment:
You need to make these guards a lot safer, since this rule was getting some 'undefined is not an object' errors on some of my files. I replaced all the guards with `lodash/get` and it seemed to fix it.